bin/download.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+var apidocs = require('node-api-docs')
+var jsonstream = require('jsonstream')
+var through = require('through2')
+var marked = require('marked')
+var mkdirp = require('mkdirp')
+var fs = require('fs')
+var path = require('path')
+
+var coredir = path.join(__dirname, '../core')
+mkdirp.sync(coredir)
+
+apidocs.json('index')
```

Comment:
Should we do something about https://nodejs.org/api/documentation.html and https://nodejs.org/api/synopsis.html?